src/openrct2/cmdline/SimulateCommands.cpp

```diff
@@ -0,0 +1,85 @@
+#pragma region Copyright(c) 2014 - 2017 OpenRCT2 Developers
```

Comment:
2018

Reply:
Ops, I'll copy the more recent copyright notice.